### PR TITLE
TP create account

### PR DESCRIPTION
**Pull request by boydjohnson**
This PR is built off of #12. To test it run `./bin/run_integration_test blockchain`. It adds the functionality to the Transaction Processor to handle a create account transaction.

### CODE REVIEW

File: transaction_creation/marketplace_transaction/common.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        dependencies=[],
+        family_name=addresser.FAMILY_NAME,
+        family_version='1.0',
+        nonce=uuid4().hex,
+        signer_public_key=signer_pubkey,
+        payload_sha512=payload_sha512)
+    return header
+
+
+class Key(object):
```
Comment (peterschwarz) [inline transaction_creation/marketplace_transaction/common.py]: You don't need to create this Key class.  You should use the sawtooth_signing.Signer class instead.  It encapsulates the context and the private key.  You can access the public key via `signer.get_public_key().as_hex()` for use in the headers.  See the updated [developer's guide](https://sawtooth.hyperledger.org/docs/core/releases/1.0.0rc3/_autogen/sdk_submit_tutorial_python.html) on this.

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: transaction_creation/marketplace_transaction/common.py (hunk 1 above)
Comment (boydjohnson) [reply transaction_creation/marketplace_transaction/common.py]: Thanks @peterschwarz . I will look into that.

File: integration_tests/blockchain/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - validator
+    command: settings-tp -vv --connect tcp://validator:4004
+
+  validator:
+    image: hyperledger/sawtooth-validator:1.0
+    expose:
+      - 4004
+    command: |
+      bash -c "
+        sawadm keygen && \
```
Comment (delventhalz) [inline integration_tests/blockchain/docker-compose.yaml]: FWIW, `\` after `&&` is unnecessary here.

File: bin/run_docker_test [hunk 3]
```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
```
Comment (delventhalz) [inline bin/run_docker_test]: Is this file necessary? What does it do that the existing `bin/run_integration_tests` does not? I would rather add a feature or two to that script than add a bunch of cruft we don't need and don't understand to the repo.

**Review by delventhalz — CHANGES_REQUESTED** [review]
We should talk offline about `bin/run_docker_test` and whether or not it should stay.

**Review by delventhalz — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]
